### ISSUE TITLE
some changes on teams for  "/pause" "/spec" players

### ISSUE THREAD

**Issue by Tsin** [DDRace/teeworlds#171]
the idea is very simple 

you are in a middle of that map  (map is quiet huge and hard) 
you failed and now you wait for other players

so in the meantime you go on "/pause" "/spec" maybe there should be team color system 

visible for this kind of players 

so when you are on pause you see all players that are in teams on diffrent colors (1 team blue, 2  team yellow, etc)

but the players that are in team dont have a color change, they see each other normaly 

this might also be a good thing for spectators 

so when on spec /pause /spec you see team players colored

but players that are in team they see their team normal way no special colors

**Comment by heinrich5991** [comment]
great idea, support that :)

**Comment by ravomavain** [comment]
good idea, will be better than just hiding other players as it was sometime proposed :-)

**Comment by Tsin** [comment]
dont know who had that idea for that "hiding players" but thats not a good idea because some players still might want to see what players in teams are doing
